src/types/api.ts

```diff
@@ -0,0 +1,648 @@
+   */
+  user_email: string;
+  /**
+   * The status of the subscription.
+   */
+  status: "on_trial" | "active" | "paused" | "unpaid" | "cancelled" | "expired";
+  /**
+   * The title-case formatted status of the subscription.
+   */
+  status_formatted: "On Trial" | "Active" | "Paused" | "Unpaid" | "Cancelled" | "Expired";
+  /**
+   * Lowercase brand of the card used to pay for the latest subscription payment.
+   */
+  card_brand: "visa" | "mastercard" | "amex" | "discover" | "jcb" | "diners" | "unionpay" | null;
+  /**
+   * The last 4 digits of the card used to pay for the latest subscription payment.
+   */
+  card_last_four: string | null;
+  /**
+   * An object containing the payment collection pause behaviour options for the subscription, if set.
+   */
+  pause: object | null;
+  /**
+   * A boolean indicating if the subscription has been cancelled.
+   */
+  cancelled: boolean;
+  /**
+   * If the subscription has a free trial, an ISO-8601 formatted date-time indicating when the trial period ends.
+   */
+  trial_ends_at: string | null;
+  /**
+   * An integer representing the day of the month on which subscription invoice payments are collected.
+   */
+  billing_anchor: number;
+  /**
+   * URLs for the customer to manage the subscription.
+   */
+  urls: {
+    /**
+     * A signed URL for managing payment and billing information for the subscription, valid for 24 hours.
+     */
+    update_payment_method: string;
+  };
+  /**
+   * An object representing the first subscription item belonging to this subscription.
+   */
+  first_subscription_item: {
+    /**
+     * ID of the subscription item.
+     */
+    id: number;
+    /**
+     * ID of the related subscription.
+     */
+    subscription_id: number;
+    /**
+     * ID of the subscription item's price.
+     */
+    price_id: number;
+    /**
+     * Quantity of the subscription item.
+     */
+    quantity: number;
+    /**
+     * Date the subscription item was created (ISO 8601 format).
+     */
+    created_at: string;
+    /**
+     * Date the subscription item was updated (ISO 8601 format).
+     */
+    updated_at: string;
+  };
+}
+
+
+interface SubscriptionObject extends BaseApiObject {
+  attributes: SubscriptionAttributes;
+}
+
+export interface SubscriptionsResponse extends BaseListResponse {
+  data: SubscriptionObject[];
+}
+
+export interface SubscriptionResponse extends BaseIndividualResponse {
+  data: SubscriptionObject;
+}
+
+
+interface StoreAttributes {
+  name: string;
+  slug: string;
+  domain: string;
+  url: string;
+  avatar_url: string;
+  plan: string;
+  country: string;
+  country_nicename: string;
+  currency: string;
+  total_sales: number;
+  total_revenue: number;
+  thirty_day_sales: number;
+  thirty_day_revenue: number;
+  created_at: string;
+  updated_at: string;
+}
+
+interface StoreObject extends BaseApiObject {
+  attributes: StoreAttributes;
+}
+
+export interface StoresResponse extends BaseListResponse {
```

Comment:
Since we updated the `BaseListResponse` & `BaseIndividualResponse` types to require a generic, we will need to update these `extends` to add a generic to specify what type you want the `included` property should be.

For example:
```suggestion
export interface StoresResponse extends BaseListResponse<unknown> {
```